vstruct/defs/elf.py

```diff
@@ -170,13 +170,18 @@ def __init__(self, bigend=False):
         self.namesz = v_uint32(bigend=bigend)
         self.descsz = v_uint32(bigend=bigend)
         self.ntype  = v_uint32(bigend=bigend)
-        self.name   = v_bytes()
-        self.desc   = vstruct.VArray()
+        self.name   = v_str()
+        self.desc   = v_str()
 
     def pcb_namesz(self):
-        self['name'].vsSetLength( self.namesz )
+        # padded to 4 byte alignment
+        namesz = self.namesz
```

Comment:
regardless of how we store the data, this alignment/padding calculation is a must-include fix.